src/foundations/icons/example-icons-size.njk

```diff
@@ -0,0 +1,64 @@
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+{{
+    onsPanel({
+        "type": 'success',
+        "id": 'success-id',
+        "iconType": 'check',
+        "iconSize": 'xxxl',
+        "body": '<h2 class="ons-u-fs-xxxl">ons-u-fs-xxxl</h2>'
```

Comment:
I think these should just say "xxxl" instead of prefixing the whole class because ultimately that is what you give to the `iconSize` param

Reply:
Yeah I agree with this, will change.